### DISCUSSION TITLE
Adding full example to readme?

### DISCUSSION THREAD

**Discussion by AnderssonPeter** [tomMoulard/fail2ban#63] [Q&A] [answered]
Hi I just found this and it sounds awesome, any chance you guys could add a example of a `docker-compose.yml` in the readme where this is used?

I'm a bit confused to I need a Fail2Ban server running or does this plugin do all the work?

**Comment by tomMoulard** [comment, marked as answer]
Hello @AnderssonPeter,

Thanks for your interest in this Traefik plugin!

> any chance you guys could add a example of a docker-compose.yml in the readme where this is used? 

Indeed there is no example on how to use this plugin, but you can find [here](https://github.com/tomMoulard/make-my-server/blob/master/traefik/docker-compose.traefik.yml) my home configuration that utilise traefik, this plugin and docker-compose.

> I'm a bit confused to I need a Fail2Ban server running or does this plugin do all the work?

This plugin is an implementation of fail2-ban made specifically to work inside traefik. Therefore in it's current form there is no need for an additional fail2-ban service. If you are looking for an alternative that uses an external service, you could take a look at the [crowdsec traefik plugin](https://plugins.traefik.io/plugins/6335346ca4caa9ddeffda116/crowdsec-bouncer-traefik-plugin).

**Reply by AnderssonPeter** [reply]
Thanks for the fast reply 🥇

**Reply by AnderssonPeter** [reply]
Just a minor thing shouldn't you add :ro to you docker socket?

**Reply by tomMoulard** [reply]
Because it's not working a one might seems. But here's some further reading that's interesting : https://medium.com/@containeroo/traefik-2-0-paranoid-about-mounting-var-run-docker-sock-22da9cb3e78c
